common/middleware/src/line_rules.rs

```diff
@@ -125,6 +125,7 @@ impl LineRules {
             redacted.extend_from_slice(&value[index..]);
         }
 
+        #[allow(clippy::question_mark)]
```

Comment:
Not sure why clippy is now complaining about this block of code but it occurs both locally and on Jenkins so I added the allow tag.